packages/devtools_server/lib/src/handlers.dart

```diff
@@ -66,20 +66,20 @@ class Api {
       case 'api/logPageView':
         return api.logPageView(request);
       default:
-        return api.notImplemented;
+        return api.notImplemented(request);
     }
   }
 
   /// Logs a page view in the DevTools server.
   ///
   /// In the open-source version of DevTools, Google Analytics handles this
   /// without any need to involve the server.
-  FutureOr<shelf.Response> logPageView(shelf.Request request) async {
-    return notImplemented;
-  }
+  FutureOr<shelf.Response> logPageView(shelf.Request request) =>
+      notImplemented(request);
```

Comment:
are we going to run into an issue because this isn't returning a Future?

Reply:
No.  It's a FutureOr, which also allows a return type of Future or a T value.